docs/expo.md

```diff
@@ -0,0 +1,45 @@
+## Guide for apps created using the Expo CLI
+
+### Overview
+Expo projects can use both third-party React Native libraries with native code and your own custom native code. Creating a development build allows you to include your specific native dependencies and customisations.
+
+- Initializing the SDKs should be done in native code inside your AppDelegate (iOS) and MainApplication (Android). 
+
+- By default, Expo projects use Continuous Native Generation (CNG). This means that projects do not have android and ios directories containing the native code and configuration. You can opt out of CNG and directly manage the code and configuration inside your android and ios directories.
+
+### Installation
+
+- To generate these directories, run `npx expo prebuild` or compile your app locally (`npx expo run android` or `npx expo run ios`).
```

Comment:
Can we request customers to build the native apps after adjusting the native code to integrate the mobile SDK?